src/lib/votesReport.ts

```diff
@@ -1,4 +1,4 @@
-import { fetchAllVotes, fetchProposal } from '../helpers/snapshot';
```

Comment:
Can we also delete https://github.com/snapshot-labs/snapshot-sidekick/blob/fix-progression-not-incremented/src/helpers/snapshot.ts#L156 it is not used anymore right?